tests/unittests/unittest_poller.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (c) 2007-2016 Contributors as noted in the AUTHORS file
```

Comment:
date range should be just 2018 - sorry but with these things it's better to be pedantic...

Reply:
Ok, thanks for updating this!